pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py

```diff
@@ -73,6 +76,59 @@ def test_identifier_with_unknown_vr():
         )
 
 
+@pytest.mark.pydicom
+def _test_identifier_is_sequence_vr():
```

Comment:
I'm not at a PC at the moment, but does this test run when it is prepended with an underscore?